webv2/web.go

```diff
@@ -777,98 +777,95 @@ func setParentTable(w http.ResponseWriter, r *http.Request) {
 }
 
 func parentTableHelper(table string, update bool) *TableInterleaveStatus {
-	tableInterleaveStatus := &TableInterleaveStatus{Possible: true}
+	tableInterleaveStatus := &TableInterleaveStatus{
```

Comment:
Parent table name not showing up in the 2nd interleave suggestion here: https://drive.google.com/open?id=1Tjhp98QK0G-8KmzYq_vVoLPRPyiB6VWJ&authuser=0&resourcekey=0-xGclQBkYSTRjrk4YNoyBcQ&usp=drive_link

Reply:
Updated. Please review it.